about.css

```diff
@@ -0,0 +1,183 @@
+/* http://meyerweb.com/eric/tools/css/reset/ 
```

Comment:
You can also take all the reset/normalize CSS code and put it in its own file as well.  Then you will not need to copy/paste it into each CSS file.